src/app/docs/passive-indicator/passive-indicator-docs.component.html

```diff
@@ -0,0 +1,153 @@
+<sky-docs-demo-page
+  moduleName="SkyProgressIndicatorModule"
+  moduleSourceCodePath="src/app/public/modules/progress-indicator/"
+  pageTitle="Passive Indicator"
```

Comment:
The current header has `Progress` in the middle, and I think I'd stick with that. It's how we refer to it everywhere else in this topic.